server/utils/queries.js

```diff
@@ -0,0 +1,34 @@
+import House from '../models/House';
+import Senate from '../models/Senate';
+
+const utils = {};
+
+/**
+ * MONGODB QUERY FOR GENERAL PURPOSE HOUSE COLLECTION QUERYING
+ * @param query === ANY AVAILABLE QUERY NAME FROM DB (OPTIONAL or NULL)
+ * @param selections === SPECIFIC DATA TO BE PASSED BACK (OPTIONAL or NULL)
+ * @param cb == CALLBACK WITH RETURNED DATA
+ */
+utils.queryHouse = (query, selections, cb) => {
+    House.find(query)
+        .select(selections)
+        .exec((err, houseDocs) => {
+           return cb(houseDocs);
+        })
+};
+
+/**
+ * QUERY FOR GENERAL PURPOSE SENATE COLLECTION QUERYING
+ * @param query=== ANY AVAILABLE QUERY NAME FROM DB (OPTIONAL or NULL)
+ * @param selections === SPECIFIC DATA TO BE PASSED BACK (OPTIONAL or NULL)
+ * @param cb == CALLBACK WITH RETURNED DATA
+ */
+utils.querySenate = (query, selections, cb) => {
```

Comment:
You see that only model name is changing for these two methods. Why can't you get a model name in the parameters and write a single method for the find?